src/spiral/experimental.clj

```diff
@@ -31,6 +31,11 @@
             (async/>! (:async-response req) shed-response)))))
     req-chan))
 
+(defn- strip-guard-prefix [path guard]
```

Comment:
Sure, I can add a docstring. And `:path-info` and `:uri` are treated differently because the servlet spec says `:path-info` should default to `nil`, and `:uri` should default to `"/"`.